deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -42,11 +43,13 @@ spec:
   customresourcedefinitions:
     owned:
     - description: PerformanceProfile is the Schema for the performanceprofiles API.
+      displayName: Performance Profile
       kind: PerformanceProfile
       name: performanceprofiles.performance.openshift.io
       version: v1alpha1
-      displayName: placeholder
-  description: Placeholder description
+  description: |2-
```

Comment:
`|2-`?

Reply:
| = keep new lines
2 = indent with two spaces
- = no new line at the end

yaml is fun ;)
https://yaml-multiline.info/